CONTRIBUTING.md

```diff
@@ -2,10 +2,19 @@
 Contributing to pydicom
 =======================
 
-This is the summary for contributing code, documentation, testing, and filing
-issues. Please read it carefully to help making the code review process go as
-smoothly as possible and maximize the likelihood of your contribution being
-merged.
+This is the guide for contributing code, documentation and tests, and for
+filing issues. Please read it carefully to help making the code review
+process go as smoothly as possible and maximize the likelihood of your
+contribution being merged.
+
+_Note:_  
+If you want to contribute new functionality, you may first consider if this 
+functionality belongs to the pydicom core, or is better suited for
+[contrib-pydicom](https://github.com/pydicom/contrib-pydicom). contrib-pydicom
+collects some convenient functionality that is using pydicom, but does not
+belong to the pydicom core. If you are unsure, create an issue where you can
+discuss this before creating a pull request.
+
 
 How to contribute
```

Comment:
We have this and the two contribution guides in the docs, should we replace this section with a link to them?

Reply:
Not sure... I would like it to be more or less self-contained, as this is the standard contribution guide GitHub expects. @darcymason , what do you think?

Reply:
Hmm, I'm not too concerned either way, but I think the principle of keeping things in one place is more important, to avoid mismatched information in future edits.  In other words, I'd prefer summary information with a link for details.

Reply:
Actually, after checking the contribution guides in the docs I think they are better written, IMO. So maybe you are right - we can leave some basic information here (including the hint about contrib-pydicom), and add a link to the documentation.